flixel/FlxGame.hx

```diff
@@ -259,6 +269,16 @@ class FlxGame extends Sprite
 	}
 	
 	/**
+	 * Sets the filter array to be applied to the game.
+	 * 
+	 * @param	filters
+	 */
+	public function setFilters(filters:Array<BitmapFilter>):Void
```

Comment:
This setter seems even more redundant, considering `FlxGame` inherits `filters` from `DisplayObject`.